src/lib/Scenes/Artwork/Components/__tests__/CommercialButtons-tests.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from "@artsy/palette"
+import { mount } from "enzyme"
+import React from "react"
+import { RelayProp } from "react-relay"
+import { CommercialButtons } from "../CommercialButtons"
+
+let artwork = {
+  slug: "abbas-kiarostami-untitled-7",
+  internalID: "5b2b745e9c18db204fc32e11",
+  is_acquireable: false,
+  is_offerable: false,
+  is_biddable: false,
+  is_inquireable: true,
+}
+
+describe("CommercialButtons", () => {
+  artwork = {
+    slug: "abbas-kiarostami-untitled-7",
+    internalID: "5b2b745e9c18db204fc32e11",
+    is_acquireable: false,
+    is_offerable: false,
+    is_biddable: false,
+    is_inquireable: true,
+  }
+  it("renders inquierable button", () => {
```

Comment:
These are weird words 😅 , but should be `inquireable` (and `acquireable` below) to match internal naming.

Reply:
Ah yeah, fixed the wording here.